StitchCore/StitchCore/Source/Core/Hmac.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
```

Comment:
Should be able to get rid of this

Reply:
Can actually get rid of this in a lot of places 👍